spec/models/redirection_spec.rb

```diff
@@ -83,4 +83,32 @@
       expect(redirection.tag_uri).to eq(id)
     end
   end
+
+  describe "#unlink" do
+    it "destroys the redirection" do
+      redirection = create(:redirection)
+
+      redirection.unlink
+
+      expect(Redirection.find_by(id: redirection.id)).to be_nil
```

Comment:
This isn't as pretty as I'd like, but:

* `expect(Redirection.count).to eq 0` fails because we always have 2 seeded redirections, for Edward and Gabe, and
* `expect(Redirection.count).to eq 2` is just letting a [Mystery Guest](https://robots.thoughtbot.com/mystery-guest) wipe their muddy shoes all over our nice clean house